catalystwan/dataclasses.py

```diff
@@ -453,8 +453,9 @@ class RadiusServer(DataclassBase):
     address: str = field(metadata={FIELD_NAME: "address"})
     auth_port: int = field(metadata={FIELD_NAME: "authPort"})
     acct_port: int = field(metadata={FIELD_NAME: "acctPort"})
-    vpn: int = field(metadata={FIELD_NAME: "vpn"})
-    vpn_ip_subnet: str = field(metadata={FIELD_NAME: "vpnIpSubnet"})
+    sourceVpn: Optional[int] = field(metadata={FIELD_NAME: "sourceVpn"})
```

Comment:
create a subclass and update the new fields in the subclass. donot change existing characteristics.
Also, these payloads should be backward compatible.

Reply:
made the change, please re-review